src/main/java/com/mindsnacks/zinc/classes/fileutils/FileHelper.java

```diff
@@ -34,27 +36,42 @@ public void unzipFile(final File originFolder,
         if (!output.exists()) {
             createDirectories(output);
 
-            final InputStream in;
-
-            try {
-                in = new BufferedInputStream(new GZIPInputStream(new FileInputStream(input)));
-            } catch (ZipException e) {
-                throw new ZincRuntimeException("Error opening gzip file: " + input.getAbsolutePath(), e);
-            }
-
+            final InputStream in = getGzippedInputStream(input);
             final ValidatingDigestOutputStream digestStream = mHashUtil.wrapOutputStreamWithDigest(new FileOutputStream(output));
             final OutputStream dest = new BufferedOutputStream(digestStream);
-            try {
-                copy(in, dest);
-            } finally {
-                dest.close();
-                in.close();
-            }
-
+            copyFile(in, dest);
             digestStream.validate(expectedHash);
         }
     }
 
+    public void unzipFileWithoutValidation(final File input,
+                                           final File output) throws IOException {
```

Comment:
This is kind of inconsistent with the previous method (the separation between folder and file names). I actually would probably have reused the previous method and make `expectedHash` optional for validation or something like that.